uwblueprint/don-efficace
# Create Donations Table with dropdown menus, filtering, and dynamically displayed data

## Notion ticket link

[Create Donations Table Component](https://www.notion.so/uwblueprintexecs/create-donations-table-component-bfbd577bcc6b419ab277e273807d5fd5?pvs=4)



## Implementation description
* We created a donations table component and a filter dropdown component. Both are included in DashboardPage.tsx (under a new /dashboard path) which fetches all the data from the endpoint. 



## Steps to test
1. Pagination: add more than 10 donations in Prisma, and try going "next" and "back."
2. Filtering: make sure the donations added are diverse. Use the dropdown menus to filter between donations.
3. Reset button: click the reset button to clear all applied filters.


## What should reviewers focus on?
* The database has been finicky to load up and required some changes in .env to load, changing ```localhost:5432``` to ```de_db:5432```
* Styling with Chakra was somewhat rigid; please let us know if there's anything in the design that should be changed.


## Checklist
